src/screens/TokenScreen.tsx

```diff
@@ -176,19 +175,29 @@ export const TokenScreen = ({ route, navigation }: Props) => {
   // Keep the description for the token up to date
   useEffect(() => {
     const updateDescription = async () => {
-      const tokens = secret.tokens ? [...secret.tokens] : []
-      const existingItemIndex = tokens.findIndex(item => item.token === token)
-      if (existingItemIndex === -1) {
+      const existingTokens = secret.tokens ? [...secret.tokens] : []
```

Comment:
I am not convinced that the new implementation is better than the original. In any case, let's please keep this PR to what the original intent is without introducing additional unrelated changes

Reply:
Understood, I'll revert these changes.